shell/chart/monitoring/grafana/index.vue

```diff
@@ -87,6 +87,9 @@ export default {
 
       return (storageClasses || []).length >= 1;
     },
+    showGrafanaResourceConfig() {
+      return this.value?.grafana?.resources?.requests && this.value?.grafana?.resources?.limits;
```

Comment:
What happens if these are not present but the user wants to set them? 

The [grafana chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana) states that `resource` by default is an empty object. I'm guessing we have something in `values` though that sets them (components are visible & populated when first going to the page). Would the user ever delete them, then want to add them later in an edit?

Reply:
Yeah - for now, it was easiest to do it this way - our chart should have this. If you delete them then  you won't get the UI. I think that is okay for now.